pkg/reporter/reporter.go

```diff
@@ -99,7 +99,7 @@ func (r *MarketplaceReporter) CollectMetrics(ctxIn context.Context) (map[MetricK
 	var resultsMapMutex sync.Mutex
 
 	if len(r.meterDefinitions) == 0 {
-		return resultsMap, []error{}, errors.Wrap(ErrNoMeterDefinitionsFound, "no meterDefs found")
+		logger.Info("no meterdefs found")
```

Comment:
Let's still return here so we don't go down and do anything else. 
```suggestion
		logger.Info("no meterdefs found")
		return resultsMap, []error{}, nil
```